django_tables2/rows.py

```diff
@@ -205,24 +213,91 @@ def items(self):
             yield (column, self.get_cell(column.name))
 
 
+class BoundPinnedRow(BoundRow):
+
+    @property
+    def attrs(self):
+        '''
+        Return the attributes for a certain pinned row.
+        Add 'pinned-row' to css class attribute
+        '''
+        row_attrs = computed_values(self._table.pinned_row_attrs, self._record)
+        cssClass = "pinned-row"
+        cssClass = " ".join([
+            self.get_even_odd_css_class(),
+            cssClass,
+            row_attrs.get('class', "")
+        ])
+        row_attrs['class'] = cssClass
+        return AttributeDict(row_attrs)
+
+    def __iter__(self):
```

Comment:
`BoundPinnedRow` inherits from `BoundRow` that has exactly the same `__iter__` method, hence I think it's unnecessary to replicate it here.